tests/app/db/test_svg_db.py

```
@@ -1,96 +1,86 @@
     mock_database_cls.assert_called_once_with({"host": "localhost"})
-    
+
     # Second call: should return cached
     db2 = get_db()
     assert db2 is mock_db_instance
     mock_database_cls.assert_called_once()
 
+
 @patch("src.app.db.svg_db.Database")
 @patch("src.app.db.svg_db.settings")
 def test_close_cached_db(mock_settings, mock_database_cls):
```

Ruff (0.15.1)

[warning] Unused function argument: `mock_database_cls`

(ARG001)